tools/deploy_webhooks.py

```diff
@@ -0,0 +1,138 @@
+import os
+import utils
+import yaml
+import deployment_options
+import textwrap
+
+log = utils.get_logger('deploy_webhooks')
+
+def main():
+    deploy_options = deployment_options.load_deployment_options()
+    if deploy_options.enable_kube_api:
```

Comment:
maybe
```python3
if not deploy_options.enable_kube_api:
    return
```
so you won't have to indent this entire logic

Reply:
also, note that this if statement will always return `True`, won't that create a problem?
Consider this example: https://gist.github.com/nmagnezi/8e307cbf0bb8c53f19000fb49a275fc4

Testing it:
```sh
$ python3 args.py --enable-kube-api xyz 
in if deploy_options.enable_kube_api statement
$ python3 args.py --enable-kube-api True
in if deploy_options.enable_kube_api statement
$ python3 args.py --enable-kube-api Flase
in if deploy_options.enable_kube_api statement
```

Reply:
I see the same pattern in a lot of Python scripts under tools.
I am not sure that this is an issue.